eth/api_backend.go

```diff
@@ -130,7 +130,8 @@ func (b *EthAPIBackend) GetEVM(ctx context.Context, msg core.Message, state *sta
 	vmError := func() error { return nil }
 
 	context := core.NewEVMContext(msg, header, b.eth.BlockChain(), nil)
-	return vm.NewEVM(context, state, b.eth.chainConfig, vmCfg), vmError, nil
+	// FIXME: Remove vmCfg param from GetEVM().
```

Comment:
Why leave it there unused?

Reply:
I'm not sure I understand the question. But `GetEVM()` is part of an interface, also implemented by LES. My idea would be to remove the `vmCfg` arg entirely, but I'd like to have a confirmation this is right direction.

Reply:
Ah I see, I didn't understand it was an interface. It just looks a bit odd to have a `vmCfg` which is totally ignored. Could we remove it from both? And if we cannot, wouldn't it be better to only ignore it if it is `nil`, to avoid surprises for callers?

Reply:
I can remove it, and create default one in LES. That will keep the current broken behavior for LES. Is that ok?

Reply:
@holiman @chfast I maintain the current behavior is not broken and my wagon PR relies on it. I still don't see why this change is necessary. So let me propose that: @chfast, could you please provide me with the specific steps that cause this issue that I can't see ? I'm looking for a use case in which `EWASMBlock` is set to 0 and your change is necessary, so that I can better understand what the problem is.

Reply:
Btw, let it be noticed that i agree with @chfast that the `vmConfig` argument should somehow be kept in agreement with what is in the chainconfig, otherwise we can end up with conflicting config. I don't agree that this is the problem that happens in this specific case, but I completely agree that it *could* happen and would be very ugly.

Reply:
This just makes the VMConfig consistent across blockchain sync, pending transaction verification and RPC calls.

To reproduce:
1. `geth --vm.ewasm=hera.so`.
2. Sync the testnet.
3. Send new ewasm transaction.

Maybe instead, you can explain why this makes wagon broken?

Reply:
I removed the `vmCfg` param from `Backend.GetEVM()`.